packages/test-runner-browserstack/src/browserstackLauncher.ts

```diff
@@ -14,7 +14,7 @@ export interface BrowserstackLauncherArgs {
 }
 
 const REQUIRED_CAPABILITIES = ['name', 'browserstack.user', 'browserstack.key', 'project', 'build'];
-const localIp = ip.address();
+const localIp = internalIpV4Sync() || '0.0.0.0';
```

Comment:
we never covered the `undefined` case, but new library has types, so the error on this line and in a similar case in `packages/test-runner-saucelabs/src/SauceLabsLauncher.ts` appeared

using fallback `0.0.0.0` seems acceptable for a test runner, although it's not a good practice in general

Reply:
checked with my colleague, and we decided it's not really a good idea
I also checked the sources of the package a bit and see that the "undefined" is more of an edge case for the browser usage, otherwise it's hard to see where else it might happen, so the next best decision is just to throw in case it's undefined

locally the types work with such if condition, hopefully the build passes too, then I merge this